src/script/auth/module/action/AuthAction.ts

```diff
@@ -185,6 +202,34 @@ export class AuthAction {
     };
   };
 
+  doGetTeamData = (teamId?: string): ThunkAction<Promise<TeamData>> => {
+    return async (dispatch, getState, {apiClient}) => {
+      dispatch(AuthActionCreator.startFetchTeam());
+      try {
+        const teamData = await apiClient.api.teams.team.getTeam(teamId ?? getState().selfState.self.team ?? '');
```

Comment:
Empty string as `teamId` doesn't make too much sense to me. Should `teamId` be a required prop?

Reply:
good idea